Gulpfile.mjs

```diff
@@ -297,6 +297,15 @@ function buildRollup(packages, targetBrowsers) {
           input,
           external,
           onwarn(warning, warn) {
+            function normalizePath(str) {
+              return typeof str == "string"
+                ? str.split(path.sep).join(path.posix.sep)
+                : str;
+            }
+
+            warning.importer = normalizePath(warning.importer);
+            warning.exporter = normalizePath(warning.exporter);
```

Comment:
Since `warning` is a Rollup object that we pass back to rollup, I'd prefer if we could avoid modifying it.
Instead, we can create an `osifyPath = str => str.split("/").join(path.sep)` function and do `warning.exporter === osifyPath("packages/babel-core/src/index.ts")` below ("osify" = "make it like the os").

Reply:
That's right! We should not modify this object.

The requested change has been completed.

But it looks a bit odd that `"@babel/helper-define-polyfill-provider" -> "@babel\helper-define-polyfill-provider"`

And it's a little inconvenient to expand in this way.When adding new conditions we need to copy and paste osifyPath.

Maybe we can deep clone the warning object, which is a small object.